zebra-state/src/service/non_finalized_state/chain.rs

```diff
@@ -195,7 +213,12 @@ impl UpdateWith<PreparedBlock> for Chain {
             // add sprout anchor and nullifiers
```

Comment:
Please delete all the "add anchor and nullifier" and "remove anchor and nullifier" comments, they don't match the code